docs/testplan/PFC_Snappi_Additional_Testcases.md

```diff
@@ -0,0 +1,65 @@
+|           04 | CONGESTION               | **DETECT CONGESTION WITH REAL-LIFE TRAFFIC PATTERN - 90% LOSSLESS and 10% LOSSY:**<br>Aim of the testcase is to ensure that congestion is detected with two ingress and single egresses of same speed and mix of lossless and lossy traffic. No losses for lossless priority traffic (Priority 3 and priority 4 streams). | - Two 100Gbps Ingress and single 100Gbps egress <br>- Repeat test with 2x400Gbps ingress and 1x400Gbps egress <br>- IMIX packet-profile <br>- Unequal traffic lossy and lossless priorities.                                           | - 90% - Lossless Priority 3 and 4 per ingress LC <br>- 10% - Lossy Background Flow [Priority 0, 1, and 2] per ingress LC.                                                                                                                                              | - Single LC - Multiple ASIC <br>- Single LC - Single ASIC <br>- Multiple LC      | 1. No packet drops for Lossy Traffic. <br>2. PFCs for lossless traffic sent from DUT ingress to traffic transmitter. No packet drops for losslesss traffic. <br>3. Expected throughput of (Lossless + lossy traffic only) on egress interface line rate of DUT.                                                                                                                                                                                                                                |
+|           05 | CONGESTION               | **DETECT CONGESTION WITH EQUAL DISTRIBUTION OF LOSSLESS AND LOSSY TRAFFIC:**<br> Purpose of the testcase is to determine that amount of congestion faced by both lossless and lossy priority traffic. No losses for priority 3 and priority 4 traffic streams.                                                            | - Two 100Gbps Ingress and single 100Gbps egress <br>- Repeat test with 2x400Gbps ingress and 1x400Gbps egress <br>- IMIX packet-profile <br>- Equal traffic for all priorities                                                         | - 24% -  Priority 3 and 4 per ingress LC <br>- 36% Background flows [Priority 0, 1, and 2] per ingress LC <br>- Enabled PFCWD and credit-watchdog on ingress and egress.                                                                                               | - Single LC - Multiple ASIC <br>- Multiple LC                                    | 1. Packet drops for Lossy Traffic. <br>2. PFCs for lossless traffic. No packet drops for losslesss traffic. <br>3. Expected throughput of egress interface line-rate (Lossless + lossy traffic only) on egress interface of DUT. <br>4. Expected close to equal throughput for all priorities.                                                                                                                                                                                                 |

```

Comment:
for the oversubscription, what are the scheduler weights configured for lossy and lossless queues.
Need to consider that to see if PFC will be generated for lossless traffic or not.